src/blocks/services/service/test/service.cypress.js

```diff
@@ -28,19 +28,53 @@ describe( 'Test CoBlocks Service Block', function() {
 	it( 'Test the service block custom classes.', function() {
 		helpers.addBlockToPost( 'coblocks/services', true );
 
-		helpers.addCustomBlockClass( 'my-custom-class', 'service' );
+		cy.get( '.wp-block-coblocks-services [data-type="coblocks/service"]:first-child' ).click( { force: true } );
+
+		helpers.upload.imageToBlock( 'coblocks/service' );
+
+		cy.get( '.block-editor-block-inspector__advanced' ).scrollIntoView().find( 'button' ).then( ( $btn ) => {
+			const isOpen = $btn.attr( 'aria-expanded' );
```

Comment:
This code is reused three times, which is a good reason to add as a helper function.

Reply:
Forgot about breaking this out into a helper. Doing that now.